index.bs

```diff
@@ -2024,21 +2031,30 @@ parameter to the {{CredentialsContainer/create()}} or {{CredentialsContainer/get
     enum AuthenticatorTransport {
         "usb",
         "nfc",
-        "ble"
+        "ble",
+        "internal"
     };
 </pre>
 
 <div dfn-type="enum-value" dfn-for="AuthenticatorTransport">
-    Authenticators may communicate with clients using a variety of transports. This enumeration defines a hint as to how clients
-    might communicate with a particular authenticator in order to obtain an assertion for a specific credential. Note that these
-    hints represent the [=[RP]=]'s best belief as to how an authenticator may be reached. A [=[RP]=] may obtain a list of
-    transports hints from some [=attestation statement formats=] or via some out-of-band mechanism; it is outside the scope of
-    this specification to define that mechanism.
+    Authenticators may communicate with clients using a variety of transports. This enumeration defines a hint as to how
+    clients might communicate with a particular authenticator. It is used in two contexts: from client to RP when
+    [creating a new credential](#createCredential), and from RP to client when when [requesting an
+    assertion](#getAssertion).
+
+    In the first context, the RP might store this information in order to present appropriate instructions to the user
+    in the future. For example, instructing the user to press an NFC authenticator to the reader. (A [=[RP]=] may also
+    obtain a list of transports hints from some [=attestation statement formats=] or via some other out-of-band
+    mechanism; it is outside the scope of this specification to define such mechanisms.)
+
+    In the second context, this is a hint representing the [=[RP]=]'s best belief as to how an authenticator may be
+    reached.
 
     <ul>
-        <li><dfn>usb</dfn> - the respective authenticator can be contacted over USB.
-        <li><dfn>nfc</dfn> - the respective authenticator can be contacted over Near Field Communication (NFC).
-        <li><dfn>ble</dfn> - the respective authenticator can be contacted over Bluetooth Smart (Bluetooth Low Energy / BLE).
+        <li><dfn>usb</dfn> - the respective authenticator can be contacted over removable USB.
+        <li><dfn>nfc</dfn> - the respective authenticator can be contacted over removable Near Field Communication (NFC).
+        <li><dfn>ble</dfn> - the respective authenticator can be contacted over removable Bluetooth Smart (Bluetooth Low Energy / BLE).
+        <li><dfn>internal</dfn> - the respective authenticator is contacted using a platform-specific transport. These authenticators are usually not removable from the platform.
```

Comment:
per https://github.com/w3c/webauthn/pull/882#issuecomment-385553226, suggest removing the three instances of "removable" in the above. 

Perhaps we should add a cross-ref to [authnr attachment](https://github.com/w3c/webauthn/pull/882#issuecomment-385553226) clarify that attachement modality and employed transports are (essentially?) orthogonal?

Reply:
Done.